content/en/docs/ambient/getting-started/deploy-sample-app/index.md

```diff
@@ -61,6 +61,14 @@ bookinfo-gateway   istio   bookinfo-gateway-istio.default.svc.cluster.local   Tr
 
 Wait for the gateway to show as programmed before continuing.
 
+{{< tip >}}
```

Comment:
We try very hard to keep the Getting Started guides to be simple, and not full of "if this then that'.  Is there any other way we can communicate this to keep it out of the "happy path"?

Reply:
That's fair. I was going to justify it by saying "we do similar stuff for GKE", but I think I hallucinated that. We definitely want to add something on in the Azure specific section

Reply:
aha! I found it: https://istio.io/latest/docs/examples/bookinfo/#start-the-application-services. Though I agree that this is not be a good precendent to follow

Reply:
The same requirement is probably true of AKS, which probably didn't exist when the page was written 🤣